### ISSUE TITLE
terraform  provider.aws: Unrecognized remote plugin message

### ISSUE THREAD

**Issue by yvusha** [QT-DevOps/DevOpsIssues#209]
when executing terraform plan/build it is showing the following error 
Error: Error asking for user input: 1 error(s) occurred:

* provider.aws: Unrecognized remote plugin message: 2|unix|/tmp/tf-plugin832199663

This usually means that the plugin is either invalid or simply
needs to be recompiled to support the latest protocol.

**Comment by raviops100** [comment]
Hi 
Can you please post what is terraform version you have installed and are you geeting this error from windows powershell ???
Thanks
Ravi

**Comment by revathi-muthana123** [comment]
Hi,

It is working for me, can you plesase share your terraform script.

Thanks & regards
Revathi

**Comment by yvusha** [comment]
Hi 

I have installed lower version of terraform terraform_0.6.6_linux_amd64, bcz of some  errors I have updated to latest version terraform_0.11.8_linux_amd64  that is the reason I got the above errors.

I got the solution : when ever we do terraform upgrade we need to remove all old files then only new version will work. This I done on Redhat 7 server now its working fine.

Sorry for the late response.
